src/styles/Search.css

```diff
@@ -0,0 +1,21 @@
+.search-input {
+  margin: auto;
+  width: 70%;
+  height: 50px;
+  border-radius: 11px;
+  font-size: 20px;
+  padding: 0 10px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.input-cont {
```

Comment:
What does this class name mean? `input-continued`? A more descriptive name would be more helpful